libs/upd/components/src/lib/data-table/data-table.component.html

```diff
@@ -114,6 +117,10 @@
           <span class="legend_color" style="background: #90ee90"></span
```

Comment:
Avoid using arbitrary colours in inline styles. Colours are almost never one-offs and are usually part of a palette. Unless it's a one-off for whatever reason, either use a pre-existing colour class, or declare a new one if it doesn't exist.